### PR TITLE
fix: user following query without feedId

### PR DESCRIPTION
**Pull request by capJavert**

### CODE REVIEW

File: src/schema/contentPreference.ts [hunk 1]
```diff
@@ -321,11 +321,11 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
         await getFeedByIdentifiersOrFail({
           con: ctx.con,
           feedIdOrSlug: args.feedId,
-          userId: ctx.userId,
+          userId: args.userId,
         });
       }
 
-      const feedId = args.feedId || ctx.userId;
+      const feedId = args.feedId || args.userId;
```
Comment (capJavert) [inline src/schema/contentPreference.ts:328]: this should never be auth user id, its a userId of requested user

Comment (omBratteng) [reply src/schema/contentPreference.ts:328]: 🙈

**Review by omBratteng — APPROVED** [review]

**Review by idoshamun — APPROVED** [review]

File: src/schema/contentPreference.ts (hunk 1 above)
Comment (omBratteng) [reply src/schema/contentPreference.ts:328]: I blame my copy-pasta

Comment (capJavert) [reply src/schema/contentPreference.ts:328]: usual suspect 😆

**Pull request merged**